ina/ina219.go

```diff
@@ -118,24 +114,23 @@ func newINA(
 		logger.Infof("using i2c address : %d", defaultI2Caddr)
 	}
 
-	maxCurrent := toNano(conf.MaxCurrent)
-	if maxCurrent == 0 {
-		switch modelName {
-		case modelName219:
-			maxCurrent = maxCurrent219
-			logger.Info("using default max current 3.2A")
-		case modelName226:
-			maxCurrent = maxCurrent226
-			logger.Info("using default max current 20A")
-		}
-	}
-
 	resistance := toNano(conf.ShuntResistance)
 	if resistance == 0 {
 		resistance = senseResistor
 		logger.Info("using default resistor value 0.1 ohms")
 	}
 
+	var maxCurrent int64
+	switch modelName {
+	case modelName219:
+		// shunt voltage for ina219 is 320mV
+		maxCurrent = toNano(float64(0.320) / fromNano(float64(resistance)))
+	case modelName226:
+		// shunt voltage for ina226 is 81.92mV
+		maxCurrent = toNano(float64(0.08192) / fromNano(float64(resistance)))
+	}
```

Comment:
[nit] can you just move these to defaults at the top of the file. They're magic numbers anyway,

Reply:
done